web/packages/selfhosted/wdio.conf.ts

```diff
@@ -22,7 +22,6 @@ if (chrome) {
         args.push("--headless");
```

Comment:
BTW shouldn't this be `"headless"`?
https://github.com/webdriverio/webdriverio/blob/933356f0539604814a8e310672f85eecb1fffce1/website/docs/Capabilities.md#run-browser-headless

Reply:
Not sure it matters, `--headless` is mentioned by https://github.com/webdriverio/webdriverio/blob/main/website/docs/Docker.md

Reply:
In this totally (not official but generated automatically) list of all chrome CLI flags, it's `--headless`. All flags are `--foo`.
https://peter.sh/experiments/chromium-command-line-switches/

Reply:
It's so inconsistent on the link I posted... 😬